### DISCUSSION TITLE
Undefined array key 'type' | FieldArgument.php

### DISCUSSION THREAD

**Discussion by EricChambersPowerDigital** [webonyx/graphql-php#1131] [Q&A] [answered]
**Thrown Error:** ErrorException: Undefined array key &quot;type&quot; in file /var/www/html/vendor/webonyx/graphql-php/src/Type/Definition/FieldArgument.php on line 89

**Error Code Location:**
<img width="821" alt="Screen Shot 2022-04-25 at 10 20 53 AM" src="https://user-images.githubusercontent.com/104379378/165131344-a992e8b9-afea-45ca-91e6-4a1c5a7b63b1.png">

**Console Log of Data Issue:**
<img width="1039" alt="Screen Shot 2022-04-25 at 10 22 41 AM" src="https://user-images.githubusercontent.com/104379378/165131756-f0533646-33cc-4e7e-8bb3-325e3732742a.png">

**Composer Package Versions:**
- "php": "^8.0"
- "laravel/framework": "^8.83"
- "webonyx/graphql-php": "^14.6.4"
- "rebing/graphql-laravel": "^8.2"

**Hack that fixes issue:**
<img width="1220" alt="Screen Shot 2022-04-25 at 10 24 48 AM" src="https://user-images.githubusercontent.com/104379378/165132012-ac08ff75-86fa-4009-aae0-c7f16ed5f9b8.png">

**Thought Process & More Information:**
The type definition is coming from the DB model class, the data type is the same as many other data types that still define the index as 'type'. I cannot see anything wrong with the model definition, relationships or anything in the code itself. Seems that the definition is some how being built incorrectly. 

Any help would be much appreciated. Thank you.

**Comment by spawnia** [comment]
This seems like an issue that is particular to either your application or `rebing/graphql-laravel`, not a bug in this library. The definition for `contact_id` just seems to be wrong.

**Comment by mfn** [comment, marked as answer]
Indeed seems more appropriate to ask over at https://github.com/rebing/graphql-laravel/discussions/categories/q-a

**Comment by EricChambersPowerDigital** [comment]
Thank you both!
